src/tds/bulk.c

```diff
@@ -760,15 +768,26 @@ tds7_bcp_send_colmetadata(TDSSOCKET *tds, TDSBCPINFO *bcpinfo)
 		 * different from BCP format
 		 */
 		if (is_blob_type(bcpcol->on_server.column_type)) {
-			/* FIXME support multibyte string */
 			len = tds_dstr_len(&bcpinfo->tablename);
-			TDS_PUT_SMALLINT(tds, len);
-			tds_put_string(tds, tds_dstr_cstr(&bcpinfo->tablename), len);
+			tds_staticin_stream_init(&r, tds_dstr_cstr(&bcpinfo->tablename), len);
+			tds_staticout_stream_init(&w, &sysname_buf, sizeof(sysname_buf));
+			res = tds_convert_stream(tds, tds->conn->char_convs[client2ucs2], to_server, &r.stream, &w.stream);
+			len_ucs2 = w.stream.buffer - sysname_buf;
```

Comment:
Maybe would be easier to use tds_convert_string/tds_convert_string_free ?

Reply:
I thought that something like `tds_convert_string` must exists, but not managed to find it.

Reply:
Rewritten using `tds_convert_string `.